test/unit/CBot/CBotToken_test.cpp

```diff
@@ -27,12 +27,12 @@ using namespace CBot;
 class CBotTokenUT : public testing::Test
 {
 public:
-    void SetUp()
```

Comment:
I chose to switch from `SetUp/TearDown` to constructor/destructor, as Google's official advice is that [the latter is preferred](https://github.com/google/googletest/blob/master/googletest/docs/FAQ.md#should-i-use-the-constructordestructor-of-the-test-fixture-or-the-set-uptear-down-function).